.travis.yml

```diff
@@ -17,7 +17,7 @@ _install: &_install
   - sudo apt install -y libsnappy-dev
   - travis_retry make bblfsh-start
   - pip install --upgrade pip cython codecov
-  - ML_MINING_SETUP_INCLUDE_TESTS=1  pip install ".[tf, pandas]"
+  - ML_MINING_SETUP_INCLUDE_TESTS=1 SKIP_SPARK_TESTS=1 pip install ".[tf, pandas]"
```

Comment:
Nope, the should be skipped unconditionally.

Reply:
Sure, I will default it to True, and leave the flag in case someone feels like debugging that hell.

Reply:
ok